README.md

```diff
@@ -241,6 +244,9 @@ A set of options is available in order to adapt the report generated.
 * `title` (`str`): Title for the report ('Pandas Profiling Report' by default).
 * `pool_size` (`int`): Number of workers in thread pool. When set to zero, it is set to the number of CPUs available (0 by default).
 * `progress_bar` (`bool`): If True, `pandas-profiling` will display a progress bar.
+* `infer_dtypes` (`bool`): 
+  * `True` by default. When `True` the `dtype` of variables are inferred by `pandas-profiling`.
```

Comment:
Please give an example, users might not directly understand otherwise.

Reply:
Actually it's complicated to show in the doc. Maybe we can include in detail with example in [adv docs](https://pandas-profiling.github.io/pandas-profiling/docs/master/rtd/pages/advanced_usage.html)

Reply:
Okay, let me try